cleaning, reorganising and renaming stuff for the process doc (#305)
enspyrco/crowdleague: .contributing/the-process.md

-   Replace *Section* below with the section name, eg. Auth

-   Create in Drive - a folder called "*Section*: Sections Planning (CL)" in "Sections Planning (CL)".  In the folder: 

    -   a Google doc called "Use Cases < *Section* (CL)" that will hold just a list of Use Cases, so we have a short, easy to read and think about, list that is the definitive set of use cases

        -   short, clear names → balance brevity with clarity 

> -   Create an Asana Project called "> *Section*" 

Reviewer: Good catch!

Author: hehe ty

> -   Use the next available Section number (denoted x. below)
> 

Reviewer: ```suggestion
```

Author: nice pickup!

Author: implemented

> -   Create in Drive - a folder called "x. *Section*: Sections Planning (CL)" in "Sections Planning (CL)".  In the folder: 

Reviewer: ```suggestion
-   Create in Drive - a folder called "*Section*: Sections Planning (CL)" in "Sections Planning (CL)".  In the folder: 
```

>     -   a Google doc called "0. Use Cases < *Section* (CL)" that will hold just a list of Use Cases, so we have a short, easy to read and think about, list that is the definitive set of use cases

Author: changed "-" to "." => keep consistent with naming across doc

Author: keen!

Author: 0 has been exterminated

Reviewer 2: ![Alt Text](https://media.makeameme.org/created/feeling-dalek-might.jpg)

    -   Sequence Diagram

> Inside Google Drive Folder "x. *Section*: Sections Planning (CL)", create:

Author: i split up asana, github, and google drive tasks from "Create: " as it gave the freedom to give context with the title, and looked alot clearer after i did it

Author: 🙏

Inside Google Drive Folder "x. *Section*: Sections Planning (CL)", create:

-   An Asana task for each use case, linked to the GitHub issue and Google doc (see below) 
-   A Models file

-   A Drive folder for each use case, called "x. *Use Case* < *Section* (CL)" and in that folder, a Google Doc for each use case: 
      -   Create a UML  - ([example](https://docs.google.com/drawings/d/1-X-aZdVrsuFItxlwkJqusOINdk0kYb9dl3fgLTQVz94/template/preview?usp=drive_web))
      
      -   Add title - "Models < x. *Use Case* < *Section* (CL)"

>     -   Models 

Author: creating the models file was still under the use case folder, when after discussion ws decided to be at the section level, so in the process doc, moved it up to so it wasnt a child of a use case, but a child of the section folder

Reviewer: Should the model have a use case tied to it since it belongs to the section instead of a use case? Maybe we should just leave the use case off of the title? "Models <  *Section* (CL)"

Author: dadoi tam. good pickup @T-Davis! ill update it!!

Author: updated :)

>     -   A User Flow:

Author: yup!!

>     -   A User Flow:

Reviewer 2: Nice!

        1. Create a folder to hold all files needed to create a user flow diagram

>             - After creating the user flow diagram, all files in this folder dont need to be looked at. So keeping them seperate will reduce clutter and make it easier to look for the files we need to look at when creating github issues and coding! 

Author: Gave a bit more context on why these files are in this folder

Author: 😄

        
        -   Google Drawing 
-   A Drive folder for each use case, called "x. *Use Case* < *Section* (CL)" and in that folder, create:

        -   UML Diagram ([example](https://docs.google.com/drawings/d/1-X-aZdVrsuFItxlwkJqusOINdk0kYb9dl3fgLTQVz94/template/preview?usp=drive_web)) 
    -   A User Flow:

    -   A User Flow 
        1. Create a folder to hold all files needed to create a user flow diagram
            - After creating the user flow diagram, all files in this folder dont need to be looked at. So keeping them seperate will reduce clutter and make it easier to look for the files we need to look at when creating github issues and coding! 
            - Add title: "User Flow Files < x. *Use Case*  < *Section* (CL)"

        1. Create a folder to hold all files needed to create a user flow

>             - Add title: "x. *Use Case* < User Flow files < *Section* (CL)"

Author: changed title from "x. *Use Case* < User Flow files < *Section* (CL)" to "User Flow Files < x. *Use Case*  < *Section* (CL)"

moving "User flow files" to the front of the title made it easier to read when in google drive as the title is so long it gets cut off.

here is before:
![drive_before](https://user-images.githubusercontent.com/49229277/102043497-aacb5b80-3e28-11eb-92f1-4051dc4a0047.PNG)

and after:
![drive_pic_after](https://user-images.githubusercontent.com/49229277/102043444-840d2500-3e28-11eb-8ba8-4436ef83b3fa.PNG)

Author: sweet!!!

>             - Add title: "Action steps < User Flow Files < x. *Use Case* < *Section* (CL)"

Author: updated name to match folder heirachy

Author: i.e action steps is a achild of user flow files which is a child of...

Author: alg!

    -   Sequence Diagram

> Inside Google Drive Folder "x. *Section*: Sections Planning (CL)", create:

Reviewer: ```suggestion
Inside Google Drive Folder "*Section*: Sections Planning (CL)", create:
```

Author: woop fixed it

      -   Create a UML  - ([example](https://docs.google.com/drawings/d/1-X-aZdVrsuFItxlwkJqusOINdk0kYb9dl3fgLTQVz94/template/preview?usp=drive_web))
      
      -   Add title - "Models < *Section* (CL)"

-   A Drive folder for each use case, called "x. *Use Case* < *Section* (CL)" and in that folder, create:

    -   A User Flow: (see: [User flow is the new wireframe](https://uxdesign.cc/when-to-use-user-flows-guide-8b26ca9aa36a) & [The How (and Why) of User Flows](https://uxdesign.cc/the-how-and-why-of-user-flows-85df776a1e2))

        1. Create a folder to hold all files needed to create a user flow diagram

>             - After creating the user flow diagram, all files in this folder dont need to be looked at. So keeping them seperate will reduce clutter and make it easier to look for the files we need to look at when creating github issues and coding! 

Reviewer: ```suggestion
            - After creating the user flow diagram, all files in this folder don't need to be looked at. So keeping them separate will reduce clutter and make it easier to look for the files we need to look at when creating github issues and coding! 
```

                    -   (action steps)

>             - After adding action steps, create "input task" in asana

Author: added "an"! 
and i can take that pr on!! I think itl help heaps, the "input task" quotation marks dont really say alot

>             - After adding action steps, create "input task" in asana

Reviewer 2: Beautiful mate!  Thanks heaps!!!

    -   A Sequence Diagram ([tutorial](https://creately.com/blog/diagrams/sequence-diagram-tutorial/), examples: [oauth2](https://developers.google.com/identity/protocols/oauth2?csw=1))  

>         -   "Sequence Diagram < x. *Use Case*  < *Section* (CL)"

Author: yes!!
